depends/builders/darwin.mk

```diff
@@ -1,23 +1,26 @@
-build_darwin_CC:=$(shell clang) --sysroot $(shell xcrun --show-sdk-path)
-build_darwin_CXX:=$(shell clang++) --sysroot $(shell xcrun --show-sdk-path)
-build_darwin_AR:=$(shell ar)
-build_darwin_RANLIB:=$(shell ranlib)
-build_darwin_STRIP:=$(shell strip)
-build_darwin_OTOOL:=$(shell otool)
-build_darwin_NM:=$(shell nm)
-build_darwin_INSTALL_NAME_TOOL:=$(shell install_name_tool)
+tools=/Library/Developer/CommandLineTools/usr/bin
+sdk=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```

Comment:
Hardcoded paths make build scripts fragile.

Reply:
Yes. I can see why this would make it fragile. This would make the build script fail when somebody install it on another path. I'll do something like: 

```
tools=$(shell xcode-select --print-path)/usr/bin
```